addons/event_sale/migrations/11.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,23 @@
+---Fields in module 'event_sale'---
+event_sale   / event.event.ticket       / event_type_id (many2one)      : NEW relation: event.type
+# Nothing to do
```

Comment:
You have to fill this field with the proper value of the existing event tickets. I think you can have it through event_id.event_type_id, but you have to be sure that this is the expected value.

Reply:
@pedrobaeza 
there is a constraint here that prevent to fill the ticket type or get the type from event.
https://github.com/OCA/OpenUpgrade/blob/11.0/addons/event_sale/models/event.py#L154
Am I right?